Library/Homebrew/extend/hash.rb

```diff
@@ -0,0 +1,11 @@
+class Hash
```

Comment:
I don't think it's easier to read and I don't think monkey patching is good practice on system-provided types.